### ISSUE TITLE
Nav bar on the blog page

### ISSUE THREAD

**Issue by dingeins** [ODataOrg/odataorg.github.io#45]

**Comment by juliapig** [comment]
We can refer to this website,add a part like "contact" if archive is not easy to realize : https://databricks.com/company/newsroom
![databricks](https://cloud.githubusercontent.com/assets/9974201/8027708/82f7650c-0dcf-11e5-93b4-84ef42a0ca32.png)

**Comment by liqian19891011** [comment]
Create pull request https://github.com/ODataOrg/odataorg.github.io/pull/55

**Comment by dingeins** [comment]
@juliapig @liqian19891011 I actually would believe the databricks usage is a little bit redundant. I'm not sure if we need them when we have a dedicated contribution page.

**Comment by liqian19891011** [comment]
@dingeins Yes. This is one way we think may work. We definitely can try other ways later. :)

**Comment by dingeins** [comment]
@liqian19891011 @juliapig I actually believe "Recent comments" would be a good content for the side. But I'm not currently sure about the fate of Disqus yet. So waiting...

**Comment by juliapig** [comment]
The fate of Disqus...haha. All right, I just think the side part needs something, but surely it should be useful.
